webdriver/tests/bidi/input/perform_actions/wheel.py

```diff
@@ -85,7 +85,13 @@ async def test_scroll_iframe(
     await bidi_session.input.perform_actions(
         actions=actions, context=top_context["context"]
     )
-    events = await get_events(bidi_session, top_context["context"])
+
+    # Chrome requires some time to process the event from the iframe, so we wait for it.
+    while True:
```

Comment:
I'm fine with that but can you please use the `AsyncPoll` helper as what we do in other cases?

Reply:
The pattern of using `AsyncPoll` is when some variable is updated based on events, and we want to wait it has some properties, like having a specific length. In case of this test, we don't have bidi events with the input data, and we have to get it manually. This getting is done in async wait, so there is no risk of having the poll blocking the transport.

Reply:
I think we probably need to have sort of timeout so that it does not loop forever if something goes wrong

Reply:
switched to `wait_for_future_safe`

Reply:
I think Async poll supports awaitables? See https://github.com/web-platform-tests/wpt/blob/077700f9850d358be790ff1fb86b79871f2ccf9a/webdriver/tests/interop/frames.py#L34C22-L34C38

Reply:
> I think Async poll supports awaitables? See https://github.com/web-platform-tests/wpt/blob/077700f9850d358be790ff1fb86b79871f2ccf9a/webdriver/tests/interop/frames.py#L34C22-L34C38

right! But it will look quite odd. Let me show

Reply:
done